client/src/components/organisms/AgendaCard.tsx

```diff
@@ -39,9 +27,10 @@ export const AgendaCard: React.FC<Props> = ({ agenda }) => {
   const switchRevealChoice = (prev: boolean) => {
     setRevealChoice(!prev);
   };
-  const totalCount: number = agenda.choices
-    .map(c => c.count)
-    .reduce((a, b) => a + b);
+  const totalCount: number = agenda.choices.reduce(
+    (acc, c) => acc + c.count,
+    0,
+  );
```

Comment:
요런 부분은 `useMemo`를 이용하면 최적화해줄수 있어요!
```suggestion
  const totalCount: number = useMemo(() => agenda.choices.reduce(
    (acc, c) => acc + c.count,
    0,
  ), [agenda.choices]);
```
이렇게 하면 `agenda.choices`가 바뀔 때만 계산됩니다.